src/locales/en.json

```diff
@@ -639,5 +639,6 @@
     "intro": "CC Search is now called Openverse and joins WordPress as an open source project.",
     "more": "{read-more} about this announcement.",
     "read": "Read more"
-  }
+  },
+  "interpunct": "•"
```

Comment:
Is it possible to add translators comments for this? It might be helpful for something this specific to give the context of where it is used without the translators having to interpret the template code.

Reply:
We don't have a good way to add comments yet sadly. Need to think about the best way to support it. I can ask some i18n folks about of we should do this though before we push

Reply:
I believed this was merely an aesthetic detail rather than something that needed translation.

Reply:
I think it might be good to allow it to be translated... for example in [Eastern Arabic, Persian and Urdu numerals](https://wikiless.org/wiki/Eastern_Arabic_numerals?lang=en) the interpunct looks very similar to the notation for `0`, "٠", which could be confusing given that this separates the timestamp from the rest of the text surrounding it.